generic-styles/index.less

```diff
@@ -28,6 +28,15 @@
       margin-inline-start: 0;
     }
   }
+  code {
+    color: #c7254e;
+    background-color: #f9f2f4;
+    border-radius: 4px;
+  }
+  [data-type="composite-page"] [data-type="exercise"] [data-type="problem"] p:first-of-type a {
```

Comment:
is this the exact same selector that cnx uses? a little worried we're gonna miss some random type

Reply:
CNX has [data-is-baked="true"] .media-body>#content [data-type="composite-page"] [data-type="exercise"] [data-type="problem"] p:first-of-type a {...}

Reply:
ok yea this is 👍 then